apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/KafkaProducerProperty.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore;
+
+import com.expediagroup.apiary.extensions.events.metastore.common.Property;
+import com.expediagroup.apiary.extensions.events.metastore.common.io.jackson.JsonMetaStoreEventSerDe;
+
+public enum KafkaProducerProperty implements Property {
+  TOPIC("topic", null),
+  BOOTSTRAP_SERVERS("bootstrap.servers", null),
+  CLIENT_ID("client.id", "CircusTrainEventDrivenEmitter"),
+  ACKS("acks", "all"),
+  RETRIES("retries", 3),
+  MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION("max.in.flight.requests.per.connection", 1),
+  BATCH_SIZE("batch.size", 16384),
+  LINGER_MS("linger.ms", 1L),
+  BUFFER_MEMORY("buffer.memory", 33554432L),
+  SERDE_CLASS("serde.class", JsonMetaStoreEventSerDe.class.getName());
+
+  private static final String PROPERTY_PREFIX = "com.expediagroup.apiary.extensions.events.metastore.";
```

Comment:
This will also need to change when you change package name if we decide to keep the prefix in our property name.